salt/modules/virt.py

```diff
@@ -1903,6 +1934,8 @@ def update(
         "disk": ["disks", "disk_profile"],
         "interface": ["interfaces", "nic_profile"],
         "graphics": ["graphics"],
+        "serial": ["serial"],
+        "console": ["console"]
```

Comment:
You may need to enhance the diffing function as done in `_diff_graphics_list()`. Be aware that libvirt will slightly change the XML when running the VM. I expect something like this to be needed:

```python
def _serial_equal(serial1, serial2):
    # TODO diff the XML fragments based only on the meaningful elements and properties

def _diff_serial_list(old, new):
    return _diff_lists(old, new, _serial_equal)
```

And surely the same for consoles... unless you can share some of that code for both since the data are fairly similar